src/test/java/org/occiware/tinom/extensions/ssh/ToRunByHand.java

```diff
@@ -0,0 +1,20 @@
+package org.occiware.tinom.extensions.ssh;
```

Comment:
Remove this file.
Unit tests might be better, with JUnit and EasyMock. Anyway, the code is simple enough...